gather_istio

```diff
@@ -106,10 +106,10 @@ function getSynchronization() {
   local pilotName
   pilotName=$(getPilotName ${namespace})
 
-  echo "Overall Envoy synchronization status for namespace ${namespace}" 2>&1 1>${logPath}/controlPlaneStatus
+  { echo "Overall Envoy synchronization status for namespace ${namespace}" > ${logPath}/controlPlaneStatus; } 2>&1
   local logPath=${BASE_COLLECTION_PATH}/namespaces/${namespace}/controlplane
   mkdir -p ${logPath}
-  oc exec ${pilotName} -n ${namespace} -c discovery -- /usr/local/bin/pilot-discovery request GET /debug/syncz 2>&1 1>${logPath}/synchronization
+  { oc exec ${pilotName} -n ${namespace} -c discovery -- /usr/local/bin/pilot-discovery request GET /debug/syncz > ${logPath}/synchronization; } 2>&1
```

Comment:
Not sure why stderr was redirected to stdout here. Why not just let the errors stay in stderr?

Reply:
Talked to @brian-avery. The intention was to redirect any error messages to the file so that we're the ones who can look at the error.